### DISCUSSION TITLE
psycopg2 install error

### DISCUSSION THREAD

**Discussion by whitedl** [bmeares/Meerschaum#64] [Q&A] [answered]
Hello. I am installing mrsm on a fresh VM. I have docker installed and confirmed that the service/daemon is running. When trying to run "stack up up -d db" I get the following error: " 💢 Failed to connect to the Docker engine."

When running "mrsm sql" there is a an error related to psycopg2 install. See the screen grab below.
![Screen Shot 2022-06-27 at 3 09 14 PM](https://user-images.githubusercontent.com/8012877/176017808-95cfa78d-f00c-468b-8af9-cde355a3fec5.png)

My Linux OS:
NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

Docker version 20.10.12, build 20.10.12-0ubuntu2~20.04.1

**Comment by bmeares** [comment, marked as answer]
Hi @whitedl, the interactive SQL CLI needs the PostgreSQL driver installed (instead of the automatically installed binary one). On Ubuntu, install the `libpq5` and `libpq-dev` packages:

```bash
sudo apt update && sudo apt install libpq5 libpq-dev -y
```

For the Docker issue: is your user in the `docker` group? It sounds like the daemon is running, but your account doesn't have privileges. You can add your user to the group with `usermod`:
```
sudo usermod -aG docker whitedl
```

I hope this helps!

**Comment by whitedl** [comment]
It was probably the docker group issue causing the connection error. Removed docker added the #USER and uninstalled meerschaum and installed all again. That seemed to take care of the issue.
________________________________
